examples/tutorials/analysis-1d/ebl.py

```diff
@@ -0,0 +1,292 @@
+notebook shows how to use these models to correct for this interaction.
+
+"""
+
+######################################################################
+# Setup
+# -----
+#
+# As usual, we’ll start with the standard imports …
+#
+
+from copy import deepcopy
+import astropy.units as u
+from astropy.coordinates import Angle, SkyCoord
+from astropy.time import Time
+from regions import CircleSkyRegion
+import matplotlib.pyplot as plt
+from gammapy.catalog import SourceCatalog4FGL
+from gammapy.datasets import SpectrumDatasetOnOff
+from gammapy.estimators import FluxPointsEstimator
+from gammapy.maps import MapAxis, RegionGeom, WcsGeom
+from gammapy.modeling import Fit
+from gammapy.modeling.models import (
+    EBL_DATA_BUILTIN,
+    EBLAbsorptionNormSpectralModel,
+    GaussianPrior,
+    PowerLawSpectralModel,
+    SkyModel,
+)
+
+######################################################################
+# Load the data
+# -------------
+#
+# We will use 6 observations of the blazars PKS~2155-304 taken in 2008 by
+# H.E.S.S. when it was in a steady state. The data have already been
+# reduced to OGIP format\ `SpectrumDatasetOnOff` following the procedure
+# :doc:`/tutorials/analysis-1d/spectral_analysis` tutorial using a
+# `ReflectedRegions` background estimation. The spectra and IRF from the
+# 6 observations have been stacked together.
+#
+# We will load this dataset as a `SpectrumDatasetOnOff` and proceed with
+# the modeling. You can do a 3D analysis as well.
+#
+
+dataset = SpectrumDatasetOnOff.read(
+    "$GAMMAPY_DATA/PKS2155-steady/pks2155-304_steady.fits.gz"
+)
+
+print(dataset)
+
+
+######################################################################
+# Model the observed spectrum
+# ---------------------------
+#
+# The observed spectrum is already attenuated due to the EBL. Assuming
+# that the intrinsic spectrum is a power law, the observed spectrum is a
+# `CompoundSpectrumModel` given by the product of an EBL model with the
+# intrinsic model.
+#
+
+
+######################################################################
+# For a list of available models, see
+# :doc:`/api/gammapy.modeling.models.EBL_DATA_BUILTIN`.
+#
+
+print(EBL_DATA_BUILTIN.keys())
+
+# define the power law
+index = 2.3
+amplitude = 1.81 * 1e-12 * u.Unit("cm-2 s-1 TeV-1")
+reference = 1 * u.TeV
+pwl = PowerLawSpectralModel(index=index, amplitude=amplitude, reference=reference)
+pwl.index.frozen = False
+# Specify the redshift of the source
+redshift = 0.116
+
+# Load the EBL model. Here we use the model from Dominguez, 2011
+absorption = EBLAbsorptionNormSpectralModel.read_builtin("dominguez", redshift=redshift)
+
+
+# The power-law model is multiplied by the EBL to get the final model
+spectral_model = pwl * absorption
+print(spectral_model)
+
+# Now, create a sky model and proceed with the fit
+sky_model = SkyModel(spatial_model=None, spectral_model=spectral_model, name="pks2155")
+
+dataset.models = sky_model
```

Comment:
Maybe remind here as a comment or in the text cell, that since this dataset has been produced by a reflected region analysis, it uses ON-OFF statistic and does not require a background model.